.github/containers/nox-cross-arch/README.md

```diff
@@ -0,0 +1,21 @@
+# Cross-Arch Testing Containers
```

Comment:
Maybe mention this, add a link to this file or something, in the  `CONTRIBUTING.md` file? And maybe also in the `README.md` we should mention that now ARM is officially supported or something?

I feel this will have very little visibility after merged, but maybe it is OK, maybe people will find out where it works or doesn't by just trying to install it. Anyway, mentioning it in case someone has a good idea :)

Reply:
Sure, I can extend the other docs too.

Reply:
I extended the contribution docs, and added a release note.

Is there a place where we specify the officially supported setups? E.g., Ubuntu 20.04, python 3.11, amd64 (and now arm64)? I could not find it in the README, and it could make sense to add one there.

Reply:
I don't think so, at some point I think we had something in the readme, but it got removed. I think the readme should be a good place to have a small section mentioning supported archs and python versions. I wonder if we could add some badge that actually gets it from the CI, but maybe for now we are fine with a static version.

Reply:
Okay, so how do we proceed here? We add a `Supported Platforms` sections to the README?

Reply:
Yeah, do it. Then we see if we find a better way.

Reply:
Done.